### ISSUE TITLE
(JS-0128) Should not have unused variables

### ISSUE THREAD

**Issue by Zxce3** [Zxce3/profile#4]
## Description
Found variables that are declared but not used anywhere.

## Occurrences
There are 3 occurrences of this issue in the repository.

See all occurrences on DeepSource &rarr; [deepsource.io/gh/Zxce3/profile/issue/JS-0128/occurrences/](https://deepsource.io/gh/Zxce3/profile/issue/JS-0128/occurrences/)

**Comment by candymd** [comment]
can i take this?

**Comment by Zxce3** [comment]
> can i take this?

@candymd yes, of course, but I haven't finished it yet, it's still a mess
